examples/monorepo-package/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "name": "docz-gatsby-monorepo",
+  "version": "2.0.0-rc.41",
```

Comment:
This should probably get a version bump?

Reply:
This is the example package, it will never be published to npm, so the version doesn't really matter.

Reply:
ah it's the version, not the dependency. My bad 🙈